cfgmgr/portmgr.cpp

```diff
@@ -192,6 +192,15 @@ void PortMgr::doTask(Consumer &consumer)
                 }
             }
 
+            if (!portOk)
+            {
+                // Port configuration is handled by the orchagent. If the configuration is written to the APP DB using
+                // multiple Redis write commands, the orchagent may receive a partial configuration and create a port
+                // with incorrect settings.
+                field_values.emplace_back("mtu", mtu);
```

Comment:
there are defaults in orchagent for these attributes. are you saying those defaults are different from this ones?

Reply:
Hi @prsunny , the problem here is the invalid admin status missing in database, which cause the port been created:
2024-09-20.02:50:54.253615|PORT_TABLE:Ethernet1000|SET|lanes:1000,1001,1002,1003|speed:100000|mtu:9100